src/app/fetchdata.js

```diff
@@ -0,0 +1,103 @@
+                            <p> <span class="description-header">Model ID:</span>${element.Model_ID} </p>
+                            <p> <span class="description-header">Model Name:</span>${element.Model_Name} </p>
+                        </div>
+                    </div>
+                    <div class="row justify-content-center">
+                        <div class="col-12 row text-center">
+                            <h3>Comments <span id="comments-count"></span></h3>
+                            <ul id="comments-ul" class="d-flex justify-center flex-col">
+                            </ul>
+                        </div>
+                        <div class="text-center col-12 commentCounter row" id="${element.Model_Name}comments"></div>
+                        <form class="col-12 row w-50">
+                            <input name= "name" type="text" id="input-name" placeholder="Your name" required>
+                            <textarea name="comment" id="comment" cols="30" rows="10" required></textarea>
+                            <span id = "alert" class="text-start"> </span>
+                            <button id="submit-comment" class="submit-comment" data="${element.Model_ID}"> Submit</button>
+                        </form>
+                    </div>
+                </div>
+                `;
+        const exit = document.querySelector('.fa-times');
+        exit.addEventListener('click', () => {
+          modal.classList.add('hidden');
+          header.classList.remove('hidden');
+          footer.classList.remove('hidden');
+          bodysection.classList.remove('hidden');
+        });
+
+        const submit = document.querySelector('.submit-comment');
+        submit.addEventListener('click', (e) => {
+          e.preventDefault();
+          const getName = document.querySelector('#input-name');
+          const getComment = document.querySelector('#comment');
+          comments.commentItem(element.Model_Name, getName.value, getComment.value);
+        });
+
+        const commentcontainer = document.getElementById(`${element.Model_Name}comments`);
+        comments.getComments(element.Model_Name)
+          .then((respo) => {
+            let cntcomments = 0;
+            respo.forEach((comm) => {
+              if (typeof comm.username === 'string') {
+                const storecomment = document.createElement('div');
+                cntcomments += 1;
+                storecomment.classList.add('col-12');
+                storecomment.innerHTML = `
+                                    <div>
+                                        userName: ${comm.username}
+                                    </div>
+                                    <div>
+                                        userComment: ${comm.comment}
+                                    </div>
+                                `;
+                commentcontainer.appendChild(storecomment);
+              }
```

Comment:
- [ ] You want to consider refreshing the comment section after you submit a new comment in your popup. ♻️ 


https://user-images.githubusercontent.com/79876959/143727466-548e99a4-14b4-4518-b5d2-4599943a3008.mp4

Reply:
Have fixed this. now it adds immediately. SOLVED